lighthouse/README.md

```diff
@@ -0,0 +1,82 @@
+# dd lighthouse custom agent check
+a google chrome lighthouse custom agent check for datadog
+
+## Overview
+
+Get metrics from [Google Chrome Lighthouse][1] as a custom agent check in your [Datadog](https://www.datadoghq.com/) app in real time to:
+* Visualize and monitor Lighthouse stats
+* Track and Audit scores for yours websites Accessibility, Best Practices, Performance, PWA, and SEO Audit scores
+
+## Setup
+
+The Lighthouse check is not included in the [Datadog Agent][2] package, so you will
+need to install it yourself.
+
+## Installation
+
+1. [Download the Datadog Agent][3].
+2. Download the [`lighthouse.py` file][4] for Lighthouse.
+3. Place it in the Agent's `checks.d` directory.
+
+To install the Lighthouse Check on your host:
+
+### Configuration
+
+1. Create a `lighthouse.d/` folder in the `conf.d/` folder at the root of your Agent's directory. 
+2. Create a `lighthouse.yaml` file in the `lighthouse.d/` folder previously created.
+3. Reference the example lighthouse [`conf.yaml.example` file][5] in this repository and copy it's contents into the `custom_lighthouse.yaml` file just created.
+4. Edit the `custom_lighthouse.yaml` file for your use case (please note: it's unclear how resource intensive this can be, a typical lighthouse report takes about 5-10seconds to generate so it may be necessary to have a higher than usual `minimum_collection_interval` )
```

Comment:
```suggestion
4. Edit the `conf.yaml` file for your use case. **Note**: It's unclear how resource intensive this can be, a typical lighthouse report takes 5-10 seconds to generate so it may be necessary to have a higher than usual `minimum_collection_interval`.
```